src/libsystemd-network/test-dhcp-client.c

```diff
@@ -532,6 +540,204 @@ static void test_addr_acq(sd_event *e) {
         xid = 0;
 }
 
+static uint8_t test_addr_bootp_reply[] = {
```

Comment:
const

Reply:
Ran a quick verification on this. The buffers do get modified when they get passed into `bootp_test_context->offer_buf` so it can't be const as currently written.